pkg/test/condition.go

```diff
@@ -27,9 +27,8 @@ func AssertConditionsMatch(t T, actual []toolchainv1alpha1.Condition, expected .
 // LastTransitionTime is ignored.
 func AssertContainsCondition(t T, conditions []toolchainv1alpha1.Condition, contains toolchainv1alpha1.Condition) {
 	for _, c := range conditions {
-		if c.Type == contains.Type {
+		if c.Type == contains.Type && c.Reason == contains.Reason {
 			assert.Equal(t, contains.Status, c.Status)
-			assert.Equal(t, contains.Reason, c.Reason)
```

Comment:
Why this change?

Reply:
not needed, was leftover from when I was adding twice the `Ready` condition. Reverted in https://github.com/codeready-toolchain/toolchain-common/pull/315/commits/8dc20d30637d503651bc415fed8c453b4c518a4e